src/hn/ui/NewsList.ui

```diff
@@ -1,23 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.22.2 -->
 <interface>
-  <requires lib="gtk+" version="3.20"/>
-  <template class="NewsList" parent="GtkBin">
-    <property name="can_focus">True</property>
-    <child>
-      <object class="GtkScrolledWindow" id="scrolled_window">
-        <property name="can_focus">False</property>
-        <property name="hexpand">True</property>
-        <property name="vexpand">True</property>
-        <property name="visible">True</property>
-        <property name="hscrollbar_policy">never</property>
-        <property name="shadow_type">in</property>
-	<child>
-          <object class="GtkBox" id="vbox">
-            <property name="orientation">vertical</property>
-	  </object>
-	</child>
-      </object>
-    </child>
+  <requires lib="gtk" version="4.0"/>
+  <template class="NewsList" parent="GtkScrolledWindow">
```

Comment:
`GtkScrolledWindow` is final - don't subclass it. Instead subclass `AdwBin` and put the `GtkScrolledWindow` inside.